src/auto_archiver/archivers/generic_archiver/truth.py

```diff
@@ -0,0 +1,31 @@
+    """
+    result = Metadata()
+    result.set_url(url)
+    timestamp = post['created_at'] # format is 2022-12-29T19:51:18.161Z
+    result.set_timestamp(datetime.datetime.strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%fZ"))
+    result.set('description', post['content'])
+    result.set('author', post['account']['username'])
+
+    for key in ['replies_count', 'reblogs_count', 'favourites_count', ('account', 'followers_count'), ('account', 'following_count'), ('account', 'statuses_count'), ('account', 'display_name'), 'language', 'in_reply_to_account', 'replies_count']:
+        if isinstance(key, tuple):
+            store_key = u" ".join(key)
```

Comment:
```suggestion
            store_key = " ".join(key)
```
unicode prefix is only needed for python2. py3 assumes unicode by default